### PR TITLE
Disclaimer modal block

### PR DESCRIPTION
**Pull request by teriyanaC**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--takeda-ihs--hlxsites.hlx.page/
- After: https://takeda-ihs-disclaimer-modal--takeda-ihs--hlxsites.hlx.page/?lighthouse=on

### CODE REVIEW

File: blocks/disclaimer-modal/disclaimer-modal.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+
+export default async function decorate(block) {
+  const response = await fetch('/drafts/teriyana/disclaimer-modal.plain.html');
+  const isModalAccepted = document.cookie.match(/\shcpModalDismiss=1;?/) !== null || window.location.href.indexOf('?bypassModal') > -1;
+  const shouldShowModal = !isModalAccepted || (document.location.href.indexOf('?showModal') > -1);
```
Comment (bstopp) [inline blocks/disclaimer-modal/disclaimer-modal.js:6]: Why the URL parameter for forcibly showing the modal?

Comment (teriyanaC) [reply blocks/disclaimer-modal/disclaimer-modal.js:6]: For testing purposes, if you accept the cookie the while testing, the modal won't show up anymore. So this is just to override it to show the modal if needed.

**Review by bstopp — CHANGES_REQUESTED** [review]
Because the Modal should always be checked, the logic to show/not needs to be moved to `scripts.js`; that can fetch the modal content from a predefined page.

Go ahead and make the static page to look up the modal content `/disclaimer` or whatever makes sense; move it out of drafts.

**Comment by teriyanaC** [comment]
> Because the Modal should always be checked, the logic to show/not needs to be moved to `scripts.js`; that can fetch the modal content from a predefined page.
> 
> Go ahead and make the static page to look up the modal content `/disclaimer` or whatever makes sense; move it out of drafts.

So do I just move the logic to show or not show and get everything else in the discalimer.js file?

File: scripts/scripts.js [hunk 2]
```diff
@@ -24,7 +25,42 @@ export const BREAKPOINTS = {
...
+  <div class="button-section">
+  <div class="agree"><p> ${config.agree}</p></div>
+  <div class="leave"><a class="link" href=" ${config.link} "> <p>${config.leave} </p></a></div>
+  </div>
+  `;
+      const acceptButn = main.querySelector('.agree');
+      acceptButn.addEventListener('click', () => {
+        const CookieDate = new Date();
+        CookieDate.setFullYear(CookieDate.getFullYear() + 5);
+        document.cookie = `hcpModalDismiss=1;path=/;myCookie=to_be_deleted;expires=${CookieDate.toUTCString()};`;
```
Comment (bstopp) [inline scripts/scripts.js:54]: Looks like you still have test data in here.

**Review by bstopp — CHANGES_REQUESTED** [review]

**Review by lwintch — APPROVED** [review]
looks good.  thanks for fixing the cookie logic!

**Review by bstopp — APPROVED** [review]

**Pull request merged**